hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -1043,6 +1043,66 @@ def hpat_pandas_series_max_impl(self, axis=None, skipna=True, level=None, numeri
     return hpat_pandas_series_max_impl
 
 
+@overload_method(SeriesType, 'prod')
+def hpat_pandas_series_prod(self, axis=None, skipna=None, level=None, numeric_only=None, min_count=0):
+    """
+    Pandas Series method :meth:`pandas.Series.prod` implementation.
+
+    .. only:: developer
+
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_prod
+
+    Parameters
+    -----------
+    axis:
+        *unsupported*
+    skipna: :obj:`bool` object
+        Exclude nan values when computing the result
+    level:
+        *unsupported*
+    numeric_only:
+        *unsupported*
+    min_count:
+        *unsupported*
```

Comment:
Anyway I propose to describe the parameters and add type of these ones.

Reply:
I have updated docstring. I do not know if it is a good approach to copy documentation from Pandas because we will have to update it. Maybe information that we reimplement Pandas function will be enough. @densmirn what do you think?

Reply:
Phenix documentation should be generated according to docstrings. I don't know details, but think it would be better to have more complete generated documentation, perhaps it doesn't have to be copied from Pandas.

Reply:
@densmirn could you, please, provide me a good example to orient on it?

Reply:
Your docsting seems pretty good with the following minor changes:
```
    self: :obj:`pandas.Series`
        input series
    axis: :obj:`int`
        Axis for the function to be applied on.
        *unsupported*
    skipna: :obj:`bool`, default :obj:`True`
        Exclude nan values when computing the result
    level: :obj:`int`, :obj:`str`, default :obj:`None`
        If the axis is a MultiIndex (hierarchical), count along a particular level, collapsing into a scalar.
        *unsupported*
    numeric_only: :obj:`bool`, default :obj:`None`
        Include only float, int, boolean columns.
        If None, will attempt to use everything, then use only numeric data.
        Not implemented for Series.
        *unsupported*
    min_count: :obj:`int`, default 0
        The required number of valid values to perform the operation.
        If fewer than min_count non-NA values are present the result will be NA.
        *unsupported*
```